tests/foreman/ui/test_errata.py

```diff
@@ -83,27 +83,26 @@ def _set_setting_value(setting_entity, value):
     setting_entity.update(['value'])
 
 
-def _org(module_target_sat):
-    org = entities.Organization().create()
-    # adding remote_execution_connect_by_ip=Yes at org level
-    entities.Parameter(
+@pytest.fixture(scope='module')
+def module_org(module_target_sat, module_entitlement_manifest_org):
```

Comment:
I would suggest to rename('module_org_with_parameter') the fixture names as this collides with the name of global `module_org`, so in future if any test from this module like to use the global `module_org` it would always end up using this local one.

Reply:
I think this is a good point, and I think I should also define a local, module-scoped manifest fixture to prevent collisions at the manifest level as well.

Reply:
I dont see the change yet made on this one ! Would you be making that change in this PR ?

Reply:
That's right, I haven't pushed any of my changes yet. I refactored the fixture stack for this module and `ui/test_hostcollection.py` locally, but I'm hitting issues that I am still debugging.

Reply:
@jyejare I pushed my changes for this. `ui/test_hostcollection.py` had a similar local fixture stack that I modified as well. Unfortunately, all of the UI tests for both errata and host collections are failing both locally and in CI, so I can't verify that my changes are working properly. The failures are primarily Selenium `NoSuchElementException` errors both locally and in CI; however, my local tests are failing to find a different element than the CI test runs. I'm a bit hesitant about keeping this change in this PR as a result.

Reply:
Sure, Even I observed there are 300+ tests are failing in CI with `NoSuchElementException` and not sure whats causing it !

So, if this stretching you a lot then lets not block your PR on this!  We can revisit later or can ask Team to visit as part of their audit/coverage improvement .